api/app/Models/User.php

```diff
@@ -1412,4 +797,64 @@ public function getFullNameAttribute()
     {
         return Str::trim($this->first_name.' '.$this->last_name);
     }
+
+    public static function scopeWhereGeneralSearch(Builder $query, ?string $searchTerm): Builder
+    {
```

Comment:
If you find this tricky to move, I'm not opposed to leaving it as is

Reply:
Yeah, if you want to take a stab at it you can :sweat_smile: I tried but it kept returning nothing in the tests :cry:

Reply:
Yeah, I'd rather spin it be spun off to another issue if desired to do. Could get a bit involved with refactoring